api_app/script_analyzers/observable_analyzers/dnsdb.py

```diff
@@ -2,28 +2,116 @@
 from urllib.parse import urlparse
 
 import requests
+from dateutil import parser as dateutil_parser
 
 from api_app.exceptions import AnalyzerRunException
 from api_app.script_analyzers import classes
 from intel_owl import secrets
 
 
 class DNSdb(classes.ObservableAnalyzer):
+    """Firesight passive DNS API
+
+    Support different server.
+    Support version 1 and 2.
+    Allow filter on rrtype, count value and timestamp on time fields.
+    Allow different query types: normal, with left or right wildcard and nameserver.
+    """
+
     def set_config(self, additional_config_params):
-        self.limit = additional_config_params.get("limit", 1000)
         api_key_name = additional_config_params.get("api_key_name", "DNSDB_KEY")
         self.__api_key = secrets.get_secret(api_key_name)
 
+        self.dnsdb_server = additional_config_params.get("server", "api.dnsdb.info")
```

Comment:
i know that the request seems boring but I would give the chance to customize the protocol